phase2/outline
# Docs: Basic html consumer.

Update the `yarn serve` / `yarn wds` to work correctly and update the example HTML file / readme to reflect current state.

## To test

- View the readme. ;)
- `yarn install`
- `yarn serve`

- visit [http://localhost:8000/dev](http://localhost:8000/dev)

It seems like this issue is also being affected by #79. As a symptom, some of the web components are not rendered, such as the `my-element` component. You can see that there is no shadow DOM when you look at those elements and they don't appear as they would in Storybook.

## Screenshot

![outline_example](https://user-images.githubusercontent.com/397902/126807030-4d8fc457-0b93-4ba2-b890-53f2457708ce.jpg)

## Notes

The `yarn serve` command was giving issues and saying we needed to either convert the `web-test-runner.config.*` file to use `exports`, rename to `mjs`, or add `module` and the `type` in `package.json`.

The rename seemed the least impacting, so I went with that.

## References

Based on feedback from #71

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/75"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
